# ObstacleDetection.msg
Header header
int32[] obstacle_type #障碍物类型
int32[] obstacle_id   #障碍物id
float32[] obstacle_l  #障碍物长
float32[] obstacle_w  #障碍物宽
float32[] obstacle_x  #障碍物位置x
float32[] obstacle_y  #障碍物位置y
float32[] obstacle_v  #障碍物速度
float32[] obstacle_q  #障碍物方向